ledgereth/transactions.py

```diff
@@ -213,7 +213,7 @@ def create_transaction(
         )
 
     # we need a gas price for a valid transaction
-    if not gas_price and not max_fee_per_gas:
+    if gas_price is None and max_fee_per_gas is None:
```

Comment:
According to types, neither of these are None-able.  And since you pointed out it's not required, I just pulled this check in #42.  Mind removing this?

Reply:
Nevermind, I'll fix it after the fact.